pkg/util/util.go

```diff
@@ -132,6 +132,36 @@ func ContainsString(stringarr []string, element string) bool {
 	return false
 }
 
+// ListDiff returns list of string which are in listA but not in listB
```

Comment:
these two functions are good.. but, think of `smart coding` here

Reply:
Added as `TODO` will be done in next PRs